src/user/user.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  Delete,
+  Put,
+  Query,
+  HttpCode,
+  HttpStatus,
+} from '@nestjs/common';
+import { UserService } from './user.service';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+
+@Controller('user')
```

Comment:
It was required in the task to use path 'v1/users'

Reply:
As far as I remember, this is not a requirement at all, but a recommendation for the possibility of versioning the application. In any case, I took this into account in the second part of the assignment.